docs/kratos/passwordless/06_code.mdx

```diff
@@ -29,37 +30,42 @@ import BrowserWindow from "@site/src/theme/BrowserWindow"
 In the registration flow below we are only presented an input field for the email address, as we have not defined any other traits
```

Comment:
nit: below there is no registration view, only direct new configuration section you have added and above there is only a sign-in screen example so the old text does not match the current layout.
<img width="982" alt="Screenshot 2024-10-08 at 08 52 18" src="https://github.com/user-attachments/assets/366cf6d8-fa3e-49b3-99d2-0f3cdbba2d1b">